### PR TITLE
EMP-884 Disk full

### PR DESCRIPTION
**Pull request by DanielUltimaker**
Added a test to verify the creation of the cluster partition.

### CODE REVIEW

File: tests/test_system/partitions_test.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        command = "sshpass -p ultimaker ssh root@" + PRINTER.printer_ip + " cat /etc/fstab"
+        df = str(FileOperations.runSubProcess(command)).split("\\n")
+        LOGGING.info(df)
+
+        res = False
+        for line in df:
+            if "/var/spool/cluster" in line:
+                res = True
+
+        self.assertTrue(res), "Cluster partition has not been found."
```
Comment (SatishBiharie93) [inline tests/test_system/partitions_test.py]: Message has to be inside of the assertion. so assertTrue(res, "Cluster partition has not been found.")

**Comment by SatishBiharie93** [comment]
It's also a good practise to squash multiple merge commits

File: tests/test_system/partitions_test.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import unittest
+import helpers.logger
+from helpers.utils import Utils
+from helpers.api_methods import ApiMethods
```
Comment (suppomi) [inline tests/test_system/partitions_test.py]: These two lines are not used.

File: tests/test_system/partitions_test.py (hunk 1 above)
Comment (DanielUltimaker) [reply tests/test_system/partitions_test.py]: Done.

**Pull request merged**